lua/octo/init.lua

```diff
@@ -408,6 +409,32 @@ function M.load_issue()
   )
 end
 
+-- This function accumulates all the taggable users into a single list that
+-- gets set as a buffer variable `taggable_users`. If this list of users
+-- is needed syncronously, this function will need to be refactored.
+-- The list of taggable users should contain:
+--   - The PR author
+--   - The authors of all the existing comments
+--   - The contributors of the repo
```

Comment:
thanks for documenting the function, that is something I need to do now that people may contribute to the plugin